docs/howtoguides.rst

```diff
@@ -36,70 +39,79 @@ Enable services
       
    Enable... <howtoguides/enable_index>
 
-Fix vulnerabilities
-===================
+Handle vulnerabilities
+======================
 
 .. include:: howtoguides/fix_index.rst
-   :start-line: 3
+   :start-line: 4
    :end-before: .. TOC
    
 .. toctree::
    :maxdepth: 2
    :hidden:
 
-   Fix vulnerabilities... <howtoguides/fix_index>
+   Handle vulnerabilities... <howtoguides/fix_index>
 
-Use the Pro Client
+Configure messages
 ==================
 
 .. include:: howtoguides/configure_index.rst
-   :start-line: 3
+   :start-line: 4
    :end-before: .. TOC
 
 .. toctree::
    :maxdepth: 2
    :hidden:
 
-   Use the Client... <howtoguides/configure_index>
+   Configure messages... <howtoguides/configure_index>
 
 Proxies
 =======
 
+The Ubuntu Pro Client supports proxy setups. These guides will help you to
+configure your proxies correctly.
+
 .. toctree::
    :maxdepth: 1
 
    Configure a proxy <howtoguides/configure_proxies>
    Configure TLS-in-TLS proxy <howtoguides/configure_tls_in_tls_proxy>
 
-How to collect logs
-===================
-
-.. toctree::
-   :maxdepth: 1
-   
-   Collect data logs for bug reporting <howtoguides/how_to_collect_logs>
-
-Corrupted lock files
-====================
-
-.. toctree::
-   :maxdepth: 1
-
-   Get rid of corrupted locks <howtoguides/get_rid_of_corrupt_lock>
-
 Pro Client with Docker
 ======================
 
+There are several Ubuntu Pro services that may be useful in a Docker image.
+
 .. toctree::
    :maxdepth: 1
 
    Enable Ubuntu Pro services in a Dockerfile <howtoguides/enable_in_dockerfile>
+   Create an Ubuntu FIPS Docker image <howtoguides/create_a_fips_docker_image.rst>
 
 Ubuntu Pro Client for Clouds
 ============================
 
+Ubuntu Pro is supported by AWS, Azure and GCP. For more details about this
+support, see :ref:`our explanation <expl-about-pro-cpc>`. These how-to guides
+will direct you in setting up your Public Cloud environment.
+
 .. toctree::
    :maxdepth: 1
 
    Create a customised Cloud Ubuntu Pro image <howtoguides/create_pro_golden_image>
    Cloud Ubuntu Pro images with FIPS updates <howtoguides/create_a_fips_updates_pro_cloud_image>
+   
+Troubleshooting
+===============
+
+We hope that your experience of using the Pro Client is an enjoyable one. You
+may occasionally need to check the version of the Pro Client, collect logs
+in order to report a bug, or release a corrupted lock on a file.
```

Comment:
Consider removing the first sentence and getting right to the point.

Reply:
Good point - done!